Airports/EGNX/SMR/Regions.txt

```diff
@@ -4,482 +4,2404 @@ smrGrass N052.48.20.236 W001.22.24.867
-N052.49.45.308 W001.20.17.619
-N052.49.45.184 W001.20.19.887
-N052.49.44.968 W001.20.32.064
-N052.49.45.074 W001.20.37.369
-N052.49.45.002 W001.20.43.411
-N052.49.45.179 W001.20.44.412
-N052.49.45.428 W001.20.45.208
-N052.49.46.033 W001.20.46.094
-N052.49.46.622 W001.20.46.775
-N052.49.46.995 W001.20.47.099
-N052.49.47.547 W001.20.47.188
-N052.49.48.189 W001.20.47.012
-N052.49.48.600 W001.20.46.660
-N052.49.49.027 W001.20.45.951
-N052.49.49.458 W001.20.44.568
-N052.49.49.622 W001.20.43.891
-N052.49.49.547 W001.20.48.697
-N052.49.46.552 W001.20.48.789
-N052.49.45.891 W001.20.48.908
-N052.49.45.409 W001.20.49.409
-N052.49.45.213 W001.20.49.763
-N052.49.44.981 W001.20.50.884
-N052.49.44.855 W001.20.51.562
-N052.49.44.837 W001.20.53.007
-N052.49.44.855 W001.20.54.009
-N052.49.44.926 W001.20.54.657
-N052.49.45.033 W001.20.55.512
-N052.49.45.300 W001.20.56.134
-N052.49.45.621 W001.20.56.518
-N052.49.45.959 W001.20.56.665
-N052.49.46.945 W001.20.56.910
-N052.49.47.913 W001.20.56.887
-N052.49.48.753 W001.20.56.741
-N052.49.49.110 W001.20.56.534
-N052.49.49.327 W001.20.56.033
-N052.49.49.648 W001.20.55.767
-N052.49.49.433 W001.20.58.428
-N052.49.48.987 W001.20.58.428
-N052.49.46.030 W001.20.58.583
-N052.49.45.550 W001.20.58.790
-N052.49.45.210 W001.20.59.229
-N052.49.44.871 W001.21.00.111
-N052.49.44.833 W001.21.01.026
-N052.49.44.573 W001.21.07.789
-N052.49.44.285 W001.21.08.979
-N052.49.43.795 W001.21.09.926
-N052.49.43.048 W001.21.10.262
-N052.49.42.156 W001.21.11.211
-N052.49.41.839 W001.21.12.587
-N052.49.41.578 W001.21.24.545
-N052.49.36.268 W001.21.24.432
-;grass1
-REGIONNAME East Midlands
-smrGrass N052.49.43.750 W001.21.05.422
-N052.49.43.445 W001.21.07.224
-N052.49.43.120 W001.21.08.030
-N052.49.42.814 W001.21.08.306
-N052.49.42.327 W001.21.07.825
-N052.49.42.111 W001.21.06.538
-N052.49.42.004 W001.21.04.717
-N052.49.41.968 W001.21.03.314
-N052.49.42.242 W001.20.45.561
-N052.49.42.441 W001.20.43.322
-N052.49.42.675 W001.20.41.470
-N052.49.42.964 W001.20.40.452
-N052.49.43.344 W001.20.40.240
-N052.49.43.723 W001.20.40.389
-N052.49.44.154 W001.20.42.365
-N052.49.44.154 W001.20.45.022
+; - GRASS
+REGIONNAME East Midlands
+smrGrass N052.48.20.236 W001.22.24.867
+N052.48.25.510 W001.16.59.106
+N052.51.03.520 W001.17.08.246
+N052.51.01.679 W001.22.25.724
+; - RUNWAY
+REGIONNAME East Midlands
+smrRunway N052.49.49.233 W001.20.58.383
+N052.49.49.230 W001.20.58.376
+N052.49.52.051 W001.18.23.991
+N052.49.52.051 W001.18.23.859
+N052.49.52.079 W001.18.22.245
+N052.49.54.020 W001.18.22.379
+N052.49.51.141 W001.21.00.138
+N052.49.49.223 W001.21.00.012
+; - ??
+REGIONNAME East Midlands
+grass N052.49.49.482 W001.20.58.330
+N052.49.50.941 W001.20.58.414
+N052.49.50.941 W001.20.58.516
+N052.49.49.479 W001.20.58.416
+; - S22
+REGIONNAME East Midlands
+smrWhite N052.49.41.937 W001.19.48.119
+N052.49.43.687 W001.19.48.204
+; - S40
+REGIONNAME East Midlands
+smrWhite N052.49.42.683 W001.20.05.280
+N052.49.42.388 W001.20.07.636
+; - Untitled_Polygon_2
```

Comment:
If you can please find what these are and give them some sort of name that will make editing the SMR easier in the future.